doc/DeveloperGuide.md

```diff
@@ -87,6 +87,38 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+### Use case: Renaming of an existing tag
+
+**MSS**
+
+1. User requests to list persons
+2. AddressBook shows a list of persons
+3. User requests to rename the tag of the specific person on the list
+4. User provides the new arguments for the tag
```

Comment:
there should be something that the addressbook does here. 
use cases capture the back and forth interaction between the system and the user.